trackpy/feature.py

```diff
@@ -223,7 +225,7 @@ def _refine(raw_image, image, radius, coords, max_iterations,
         for iteration in range(max_iterations):
             off_center = cm_n - radius
             if walkthrough:
-                print_update(off_center)
+                logger.info('%f', off_center)
```

Comment:
Could you put some more words here here describing this?  something like `logger.info('off center: %f', off_center)`

Is there a plan do ditch `walkthrough` and make these debugging-level logging calls eventually?